### PR TITLE
Test for Analytics link on Instructor Dashboard

### PR DESCRIPTION
**Pull request by raeeschachar**

### CODE REVIEW

**Comment by raeeschachar** [comment]
@benpatterson @Muddasser @muhammad-ammar
Please review.

File: regression/pages/lms/instructor_dashboard.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+from edxapp_acceptance.pages.lms.instructor_dashboard import (
+    InstructorDashboardPage
+)
+from regression.pages.lms import BASE_URL
+
+
+class InstructorDashboardPageExtended(InstructorDashboardPage):
+    """
+    This class is an extended class of Instructor Dashboard Page,
+    where we add methods that are different or not used in
```
Comment (benpatterson) [inline regression/pages/lms/instructor_dashboard.py]: I think just the first line of docstring is fine. If you want, you could say something about how this has extra navigation that's important to e2e tests.

File: regression/pages/lms/instructor_dashboard.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    InstructorDashboardPage
+    """
+    @property
+    def url(self):
```
Comment (benpatterson) [inline regression/pages/lms/instructor_dashboard.py]: Is this the same as the base class? If so, you can just use super and you don't need to re-declare it here.

File: regression/pages/lms/instructor_dashboard.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        """
+        self.q(css='[data-section="instructor_analytics"]').click()
+        # Click initiates an ajax call
+        self.wait_for_ajax()
+
+    def click_edx_insights(self):
+        """
+        Clicks edX Insights link on Analytics tab
+        """
+        self.q(css='p em a').click()
```
Comment (benpatterson) [inline regression/pages/lms/instructor_dashboard.py]: Is this clickable if we are _not_ on the analytics tab?

File: regression/tests/lms/test_instructor_dashboard.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        self.instructor_dashboard = InstructorDashboardPageExtended(
+            self.browser,
+            get_course_key(course_info)
+        )
+        self.course_page = CourseInfoPageExtended(
+            self.browser,
+            get_course_key(course_info)
+        )
+
+        self.dashboard_page.select_course('Manual Smoke Test Course 1 - Auto')
```
Comment (benpatterson) [inline regression/tests/lms/test_instructor_dashboard.py]: Should the course be a var that's declared somewhere else?

File: regression/pages/lms/instructor_dashboard.py [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+    def click_analytics_tab(self):
+        """
+        Clicks Analytics tab on Instructor Dashboard
+        """
+        self.q(css='[data-section="instructor_analytics"]').click()
+        # Click initiates an ajax call
+        self.wait_for_ajax()
+
+    def click_edx_insights(self):
```
Comment (benpatterson) [inline regression/pages/lms/instructor_dashboard.py]: I think you should rename this to something like `get_insights_title_text` or something like that.

File: regression/pages/lms/instructor_dashboard.py (hunk 1 above)
Comment (raeeschachar) [inline regression/pages/lms/instructor_dashboard.py]: Done

File: regression/pages/lms/instructor_dashboard.py (hunk 2 above)
Comment (raeeschachar) [inline regression/pages/lms/instructor_dashboard.py]: Actually the base URL is different.

File: regression/pages/lms/instructor_dashboard.py (hunk 5 above)
Comment (raeeschachar) [inline regression/pages/lms/instructor_dashboard.py]: Done

File: regression/pages/lms/instructor_dashboard.py (hunk 3 above)
Comment (raeeschachar) [inline regression/pages/lms/instructor_dashboard.py]: Nope it won't be clickable. This is a unique selector. 
`<p>
<em>
For analytics about your course, go to
<a target="_blank" href="https://stage-insights.edx.org/courses/course-v1:ArbiRaees+AR-1000+fall">edX Insights</a>
.
</em>
</p>`

File: regression/tests/lms/test_instructor_dashboard.py (hunk 4 above)
Comment (raeeschachar) [inline regression/tests/lms/test_instructor_dashboard.py]: Done.

**Comment by raeeschachar** [comment]
@benpatterson @Muddasser 
Comments addressed. Please review.

**Comment by benpatterson** [comment]
No need to change the link. I was hoping it wouldn't be clickable. :)

:+1:

**Comment by Muddasser** [comment]
👍

**Pull request merged**